### PR TITLE
[NavigatorIOS] Allow translucent on NavigatorIOS

### PR DESCRIPTION
**Pull request by ojkelly**
Hi,

I've updated the NavigatorIOS component to allow setting the translucent property.

usage is:
```
            <NavigatorIOS
                translucent={false}
            />
```

This is my first contrib to react-native, so apologies if I've missed something.

Cheers,
Owen

### CODE REVIEW

**Comment by brentvatne** [comment]
@ojkelly - this looks good to me, can you add an example to UIExplorer? :smile:

File: React/Views/RCTNavItem.h [hunk 1]
```diff
@@ -22,6 +22,8 @@
 @property (nonatomic, strong) UIColor *tintColor;
 @property (nonatomic, strong) UIColor *barTintColor;
 @property (nonatomic, strong) UIColor *titleTextColor;
+@property (nonatomic, assign) BOOL translucent;
+
```
Comment (a2) [inline React/Views/RCTNavItem.h]: Nit: remove newline

**Comment by a2** [comment]
Looks good as well. If you can remove that one newline (sorry, OCD :wink:) and add an example as @brentvatne suggests, I'll merge this. :)

**Comment by ojkelly** [comment]
Hey @a2 sorry for the delay, I was away.

I've removed that newline, and added an example. After looking at both NavigatorIOS examples, I thought NavigatorIOSColorsExample.js made the most sense to put it. I can move it if we need though.

**Pull request closed without merging**